small_grants.md

```diff
@@ -0,0 +1,153 @@
+@def title = "SciML Small Grants Program Current Project List"
+@def tags = ["home", "sciml", "diffeq"]
+
+# SciML Small Grants Program Current Project List
+
+The following is the current project list for the SciML Small Grants Program.
+
+## Rules and Regulations
+
+The small grant projects are decided by the SciML Steering Council and candidates
+can choose to take on projects from the project list. This is similar to 
+"bounty programs" seen in other open source environments, though it is driven 
+by SciML in a manner that is designed to give the better outcomes to both 
+contributors and maintainers of the project. In order to remove a hostile 
+competitive atmosphere, **candidates must declare to the committee interest 
+before solving the chosen issue** and, upon approval by the selection committee, 
```

Comment:
Information on how to declare to the committee should be here as well, or at least a link to that info.  

It would also be nice to have an ongoing project board for this, so that people can see not only current opportunities but also a real-time update of which projects have been taken or not taken.

Reply:
> It would also be nice to have an ongoing project board for this, so that people can see not only current opportunities but also a real-time update of which projects have been taken or not taken.

Yes clarified.